.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - '8'
+  - '10'
```

Comment:
i'm not super familiar with travis. does this mean it will run the tests 2 times on both node 8 and node 10?

At this point, I think we're fine just supporting the latest version of node. Would it be easier to just use 10? I'm not sure what the normal convention is.

Reply:
yeah, it'll run two sets of tests. i'll make it so that it always runs `stable`. i can also run it on multiple browsers if you want, if you think there's a chance for cross-platform problems

Reply:
Oh, interesting. I think browser compatibility is more a concern at this point than Node compatibility. Maybe test on the latest Chrome and Safari for both desktop and mobile? @adarob do you have thoughts?

Reply:
Oof, I don't think Travis supports mobile (or Windows, yet), but for now I can test on Chrome/FF on Linux, and Safari on OSX.

Reply:
Oh, i just realized none of the tests are browser tests -- they're just in node, so the browsers don't actually matter. 🤦‍♀️